docs/catalog.py

```diff
@@ -22,6 +22,31 @@ def custom_walk(top):
             yield entry
 
 
+def all_subtypes_of_artifact(artifact):
```

Comment:
perhaps add type hints? is `artifact` an artifact type or an actual artifact?

Reply:
Yes, I was mysterious more than absolutely needed.. The  idea is to pull out from the main artifact in your `make_content(artifact, label, all_labels):` all the inner 
**"type" : "artifact (typically, operator) name as shows in the catalog"**
and to add explanations also on these.  See [example](https://unitxt.readthedocs.io/en/latest/catalog.cards.winogrande.m.html) where not only the outer "task_card" is now explained.

Reply:
nice! very useful..